src/pseudopeople/noise_scaling.py

```diff
@@ -38,6 +38,15 @@ def scale_nicknames(data: pd.DataFrame, column_name: str) -> float:
     return 1 / proportion_have_nickname
 
 
+def scale_copy_from_household_member(data: pd.DataFrame, column_name: str) -> float:
+    copy_column = data[metadata.COPY_HOUSEHOLD_MEMBER_COLS[column_name]]
+    eligible_idx = copy_column.index[(copy_column != "") & (copy_column.notna())]
+    proportion_eligible = len(eligible_idx) / len(data)
+    if proportion_eligible == 0.0:
+        return 0.0
+    return 1 / proportion_eligible
```

Comment:
This is scaling the noise proportion just like we did for nicknames. Similar to how we have a dataset where we are choosing a certain number of rows for our noise level, we are choosing more of the eligible rows to noise due to those who are ineligible from PRL.

Reply:
I think maybe it is confusing because it looks similar to getting the rows that have NAs like in `get_index_to_noise` but we need to scale it based on NAs.